diff_cover/__init__.py

```diff
@@ -1,5 +1,10 @@
 from __future__ import unicode_literals
 
+import pluggy
```

Comment:
The plugin implementation is closely based on this excellent [example](https://pluggy.readthedocs.io/en/latest/#a-complete-example) from the Pluggy documentation.